src/test/mod.rs

```diff
@@ -137,8 +140,11 @@ impl TestEnvironment {
     fn base_config(&self) -> Config {
         let mut config = Config::from_env().expect("failed to get base config");
 
+        // create index directory
+        fs::create_dir_all(config.registry_index_path.clone()).unwrap();
+
         // Use less connections for each test compared to production.
-        config.max_pool_size = 2;
+        config.max_pool_size = 4;
```

Comment:
Why were these changes needed?

Reply:
the test was timing out to get a new connection

Reply:
Ok, and creating the index directory?

Reply:
also failed without, also this is needed for the local setup too ( as seen in the [readme](https://github.com/rust-lang/docs.rs#getting-started))